run/shared/download_data.py

```diff
@@ -243,77 +232,70 @@ def get_run_info():
     return run_info
 
 
-def get_grid_suffix(
-        resolution
-):
+def get_grid_suffix(resolution):
     """
     Given a model resolution, returns the grid filename suffix.
 
-    Args:
-    -----
-    resolution : str
-        The grid resolution (read from geoschem_config.yml)
+    Args
+    resolution : str : Grid resolution (from geoschem_config.yml)
 
-    Returns:
-    --------
-    suffix : str
-        The corresponding filename suffix
+    Returns
+    suffix     : str : The corresponding filename suffix
     """
-    if "4.0x5.0" in resolution:
-        return "4x5"
-    if "2.0x2.5" in resolution:
-        return "2x25"
-    if "0.5x0.625" in resolution:
-        return "05x0625"
-    return "025x03125"
-
-
-def get_nest_suffix(
-        longitude
-):
+    mapping = {
+        "4.0x5.0": "4x5",
+        "2.0x2.5": "2x25",
+        "0.5x0.625": "05x0625",
+        "0.25x0.3125": "025x03125"
+    }
+    for (key, value) in mapping.items():
+        if key in resolution:
+            return value
+    return "0125x015625"
```

Comment:
Why isn't 0.125x0.3125 included in the mapping, with an error thrown if the resolution does not match any of the supported options?